<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtugoosecfg</class>
 <widget class="QWidget" name="dtugoosecfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GOOSE通信配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>570</width>
       <height>420</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>570</width>
       <height>420</height>
      </size>
     </property>
     <property name="title">
      <string>GOOSE配置</string>
     </property>
     <widget class="QPushButton" name="btn_update">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>读取配置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>下发配置</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>271</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>N侧配置</string>
      </property>
      <widget class="QLabel" name="TXT1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>N侧 1节点</string>
       </property>
      </widget>
      <widget class="QLabel" name="TXT2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>N侧 2节点</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkN3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="TXT10">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>APPID</string>
       </property>
      </widget>
      <widget class="QLabel" name="TXT3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>N侧 3节点</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkN2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="TXT9">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>APPID</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinN1">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkN1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinN2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinN3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>170</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QLabel" name="TXT11">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>APPID</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>551</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>本机配置</string>
      </property>
      <widget class="QSpinBox" name="spin_cur">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>72</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QLabel" name="TXT8">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>网卡</string>
       </property>
      </widget>
      <widget class="QComboBox" name="com_net">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>eth0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>eth1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>eth2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>eth3</string>
        </property>
       </item>
      </widget>
      <widget class="MacAddress" name="macLocal" native="true">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="TXT4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>38</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>组播MAC地址</string>
       </property>
      </widget>
      <widget class="QLabel" name="TXT9_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>APPID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>注:APPID,最好在1000以上,且不可重复</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>261</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>M侧配置</string>
      </property>
      <widget class="QSpinBox" name="spinM3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkM3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinM1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QLabel" name="TXT7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>M侧 3节点</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkM2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="TXT14">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>APPID</string>
       </property>
      </widget>
      <widget class="QLabel" name="TXT6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>M侧 2节点</string>
       </property>
      </widget>
      <widget class="QLabel" name="TXT5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>M侧 1节点</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinM2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkM1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="TXT13">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>APPID</string>
       </property>
      </widget>
      <widget class="QLabel" name="TXT12">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>APPID</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MacAddress</class>
   <extends>QWidget</extends>
   <header location="global">dtumacaddress.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_update</sender>
   <signal>clicked()</signal>
   <receiver>dtugoosecfg</receiver>
   <slot>updatecfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>dtugoosecfg</receiver>
   <slot>savecfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatecfg()</slot>
  <slot>savecfg()</slot>
 </slots>
</ui>
